tests/sentry/api/endpoints/test_organization_events_trends_v2.py

```diff
@@ -0,0 +1,268 @@
+                    "field": ["project", "transaction"],
+                    "query": "event.type:transaction",
+                    "project": self.project.id,
+                },
+            )
+
+        assert response.status_code == 200, response.content
+
+        events = response.data["events"]
+        result_stats = response.data["stats"]
+
+        assert len(events["data"]) == len(mock_trends_result)
+        assert events["data"] == mock_trends_result
+
+        assert len(result_stats) > 0
+        assert len(result_stats.get(f"{self.project.slug},foo", [])) > 0
+
+    @mock.patch("sentry.api.endpoints.organization_events_trends_v2.get_trends")
+    def test_simple_with_no_trends(self, mock_get_trends):
+        mock_trends_result = []
+        mock_get_trends.return_value = {"data": mock_trends_result}
+
+        with self.feature(self.features):
+            response = self.client.get(
+                self.url,
+                format="json",
+                data={
+                    "end": iso_format(self.now),
+                    "start": iso_format(self.now - timedelta(days=1)),
+                    "interval": "1h",
+                    "field": ["project", "transaction"],
+                    "query": "event.type:transaction",
+                    "project": self.project.id,
+                },
+            )
+
+        assert response.status_code == 200, response.content
+
+        events = response.data["events"]
+        result_stats = response.data["stats"]
+
+        assert len(events["data"]) == len(mock_trends_result)
+        assert events["data"] == mock_trends_result
+
+        assert len(result_stats) == 0
+
+    @mock.patch("sentry.api.endpoints.organization_events_trends_v2.get_trends")
+    def test_simple_with_transaction_query(self, mock_get_trends):
+        mock_trends_result = []
+        mock_get_trends.return_value = {"data": mock_trends_result}
+
+        self.store_performance_metric(
+            name=TransactionMRI.DURATION.value,
+            tags={"transaction": "bar"},
+            org_id=self.org.id,
+            project_id=self.project.id,
+            value=2,
+            hours_before_now=2,
+        )
+
+        with self.feature(self.features):
+            response = self.client.get(
+                self.url,
+                format="json",
+                data={
+                    "end": iso_format(self.now),
+                    "start": iso_format(self.now - timedelta(days=1)),
+                    "interval": "1h",
+                    "field": ["project", "transaction"],
+                    "query": "event.type:transaction transaction:foo",
+                    "project": self.project.id,
+                },
+            )
+
+        trends_call_args_data = mock_get_trends.call_args[0][0]["data"]
+        assert len(trends_call_args_data.get(f"{self.project.slug},foo")) > 0
+        assert len(trends_call_args_data.get(f"{self.project.slug}, bar", [])) == 0
```

Comment:
```suggestion
        assert len(trends_call_args_data.get(f"{self.project.slug},bar", [])) == 0
```
Should there be no space here?